modules/images/dominant-color-images/hooks.php

```diff
@@ -62,7 +62,9 @@ function dominant_color_update_attachment_image_attributes( $attr, $attachment )
 		if ( empty( $attr['style'] ) ) {
 			$attr['style'] = '';
 		}
-		$attr['style'] .= '--dominant-color: #' . esc_attr( $image_meta['dominant_color'] ) . ';';
+		$style_attribute = $attr['style'];
+		$attr['style']   = '';
+		$attr['style']  .= '--dominant-color: #' . esc_attr( $image_meta['dominant_color'] ) . ';' . $style_attribute;
```

Comment:
```suggestion
		$attr['style']   = '--dominant-color: #' . esc_attr( $image_meta['dominant_color'] ) . ';' . $style_attribute;
```

Would this not work?